Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -59,6 +59,7 @@ def runSelected(opt):
                      135.4, #Run 2 Zee ttbar
                      10021.0, #2017 tenmu
                      10024.0, #2017 ttbar
+                     10424.0, #2017 NewPix ttbar
```

Comment:
@cvuosalo it was easiest just to shift all the 2023 workflows to start with 20000 and let the 2017 workflows use 10000. All of the intermediate numbers are generated automatically based on the list of generator fragments. If we start retiring workflow numbers manually, Configuration/PyReleaseValidation will become totally unmaintainable.